pkg/plugin/packetparser/_cprog/packetparser.c

```diff
@@ -237,7 +237,7 @@ int endpoint_ingress_filter(struct __sk_buff *skb)
 	// So ingress on host is egress on endpoint and vice versa.
 	parse(skb, FROM_ENDPOINT);
 	// Always return 0 to allow packet to pass.
-	return 0;
+	return -1;
```

Comment:
can we include `linux/pkt_cls.h` and return `TC_ACT_UNSPEC` instead of -1?

Reply:
Retina doesn't seem to assume kernel headers are installed so I followed the approached taken for e.g. `ETH_P_IP` and copied the definition of `TC_ACT_UNSPECIFIED` into `packetparser.h`.